scripts/testnet/batchSettleEcoTestNet.ts

```diff
@@ -19,6 +19,8 @@ import {
   networkIds,
   networks,
   actors,
+  settlementTypes,
```

Comment:
in the scripts folder we should have one deploy directory and one testing directory. Deploy handles all deploys, in files clearly labeled for mainnet or testnet. Testing should contain subdirectories for mainnet and testnet, and those should contain the test files.